gateway/api/localauth/register.go

```diff
@@ -83,16 +83,17 @@ func Register(c *gin.Context) {
 		Name:   adminGroupName,
 	}
 	trackClient := analytics.New()
-	trackClient.MergeIdentifiedUserTrack(org.ID, user.Email, analytics.EventSingleTenantFirstUserCreated, nil)
 	trackClient.Identify(&types.APIContext{
 		OrgID: org.ID,
 		OrgName: org.Name,
 		UserName: user.Name,
 		UserID: user.Email,
+		UserAnonSubject: org.ID,
```

Comment:
Are you sending the subject from an anonymous user passing the organization ID. Is it a mistake or it's intentional?

Reply:
it's intentional in this case because when the default org is created, that's the only identifiable information to later relate to the user signing up. It's weird, but I couldn't find an easier solution. If you have anything, please let me know

Reply:
You're not using the `UserAnonSubject` attribute in the `Identify` method and you already have this information in the organization ID. Would you mind explaining your intention?